drivers/memc/Kconfig.stm32

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
+
+config MEMC_STM32
+	bool "Enable STM32 Flexible Memory Controller (FMC)"
+	depends on SOC_SERIES_STM32H7X
+	help
+	  Enable STM32 Flexible Memory Controller.
+
+config MEMC_STM32_SDRAM
+	bool "Enable STM32 FMC SDRAM controller"
+	depends on MEMC_STM32
+	select USE_STM32_LL_FMC
```

Comment:
@erwango selection of LL_FMC is still required (HAL_SDRAM depends on it, but it is not specified on Kconfig)